### ISSUE TITLE
Elementary OS support

### ISSUE THREAD

**Issue by vitornogueira** [rafaelstz/Macanse#3]
Great work!
Would have the possibility of creating a theme for Elementary OS?

**Comment by rafaelstz** [comment]
Yes of course, today do not use the **Elementary OS**, but can help develop the project. Thanks for the suggestion! :+1: :smile:

**Comment by vitornogueira** [comment]
nice :smile:
